src/main/java/com/google/sps/servlets/quizTiming.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List; 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+@WebServlet("/quizes")
+public class quizTiming extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Entity quiz = new Entity("Quiz");
+    quiz.setProperty("timestamp", System.currentTimeMillis());
+    
```

Comment:
So, if I understand correctly, this servlet creates a new quiz, right? It makes an assumption that there is no existing quiz. However, based on my comments in the getDailyQuiz.java file, we could likely combine this logic into a single servlet. 

Additionally, is this servlet being used? I don't see any new references to it in this PR.